test/images/agnhost/dns/common.go

```diff
@@ -51,17 +51,17 @@ var CmdEtcHosts = &cobra.Command{
 	Run:   printHostsFile,
 }
 
-func printDNSSuffixList(cmd *cobra.Command, args []string) {
```

Comment:
These functions are used by cobra. And the form of these fuctions must be "func(cmd *Command, args []string)". So we shouldn't remove the "unused" params.